### DISCUSSION TITLE
Getting error while using social login in alchemy account kit.

### DISCUSSION THREAD

**Discussion by harshitaRajput18** [alchemyplatform/aa-sdk#1138] [Issues]
I'm using alchemy account kit in my react DApp, but getting issue while social login. I followed all the steps mentioned in documentation.  I'm attaching the screenshot for reference 
![Screenshot from 2024-11-08 11-52-52](https://github.com/user-attachments/assets/2eddcf02-63ce-40f1-b97f-e49855025710)

**Comment by kiknaio** [comment]
Hey,
I'm having same issue and seems like it's not working on localhost. 
- Have you created Google API key and provided it to Alchemy Embedded Account section? That's necessary step to make Google Auth work.
- Have you tried to deploy it somewhere other than localhost to try if it's working?

**Reply by harshitaRajput18** [reply]
I have tried it on server but it's not working
Alchemy account kit works absolutely fine with next.js but in my case it doesn't support react.

**Reply by moldy530** [reply]
hey! we've got a PR up to address this IDB error. we should have this in today, sorry about that!

**Reply by moldy530** [reply]
Are there any failed network calls in your network tab?

**Reply by harshitaRajput18** [reply]
![Screenshot from 2024-11-18 10-31-44](https://github.com/user-attachments/assets/8748705e-3267-4ee3-a986-0fb01fcb216d)
Updated all the dependencies but facing same issue again.

![image](https://github.com/user-attachments/assets/850d8ee2-123a-4aa9-b18b-4e2949576704)

**Reply by moldy530** [reply]
Can you confirm the version you're on is `v4.4.1` for all packages?

Also, the `Buffer` is not defined error might be the real culprit here. You might need to add a `buffer` polyfill to your build. If you're using vite this is a good option: https://www.npmjs.com/package/vite-plugin-node-polyfills

**Reply by harshitaRajput18** [reply]
Thank you for your support, the issue is resolved. It's because of buffer.
